robot1:
  patrols: 4
  route: multi-exp_1.csv
  start_node: 0
  

robot2:
  patrols: 8
  route: straightpat.csv
  start_node: 0


robot3:
  patrols: 4
  route: multi-exp_2.csv
  start_node: 0

trialNumber: 1
timeThresholdLow: 2
timeThresholdHigh: 6
speed: fast
usePlasticity: true